### PR TITLE
Add brazilian portuguese translation

### PR DESCRIPTION
**Pull request by jeangraff30**
credits: @jeangraff30 and Marcos Almeida

### CODE REVIEW

**Comment by idoybh** [comment]
wtf are there credits in the PR title? Please squash this mess and give the proper author authorship. Thanks

**Comment by jeangraff30** [comment]
Sorry

File: res/values-pt-rBR/derpquest_strings.xml [hunk 1]
```diff
@@ -0,0 +1,956 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2017-2019 The PixelDust Project
```
Comment (idoybh) [inline res/values-pt-rBR/derpquest_strings.xml]:
```suggestion
     Copyright (C) 2020 DerpFest
```

File: res/values-pt-rBR/derpquest_strings.xml [hunk 2]
```diff
@@ -0,0 +1,956 @@
...
+    <string name="lockscreen_clock_default_accent_hour">Default (accent hours)</string>
+    <string name="lockscreen_clock_sammy">Sammy</string>
+    <string name="lockscreen_clock_sammy_bold">Sammy (negrito)</string>
+	<string name="lockscreen_clock_sammy_accent">Sammy (destaque)</string>
+    <string name="lockscreen_clock_sammy_accent_hour">Sammy (accent hour)</string>
+    <string name="lockscreen_clock_sammy_accent_min">Sammy (accent minute)</string>
+    <string name="lockscreen_clock_text">Texto (Android 9)</string>
+    <string name="lockscreen_clock_text_alt">Texto (Android 10)</string>
+    
+	<!-- Lockscreen Date styles -->'
```
Comment (idoybh) [inline res/values-pt-rBR/derpquest_strings.xml]:
```suggestion
	<!-- Lockscreen Date styles -->
```

**Review by idoybh — CHANGES_REQUESTED** [review]

File: res/values-pt-rBR/seekbar_preference_strings.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/**
+ * Copyright (C) 2017 AICP
```
Comment (idoybh) [inline res/values-pt-rBR/seekbar_preference_strings.xml]:
```suggestion
 * Copyright (C) 2020 DerpFest
```

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
> Sorry

Awaiting either your reply with proper github usernames or your force-push with proper authorship. 
P.S: I fixed your file, make sure to pull that

**Comment by jeangraff30** [comment]
The person who helped me with the translation doesn't have a github, but I left his name in the first PR comment, thanks for the corrections

**Comment by idoybh** [comment]
> The person who helped me with the translation doesn't have a github, but I left his name in the first PR comment, thanks for the corrections

Oh if that's the situation I'll make sure he is mentioned in the commit msg. Thanks

**Comment by idoybh** [comment]
Will merge soon. Thank you for your hard work translating :)

**Comment by jeangraff30** [comment]
Thanks for the hard work on the rom

**Pull request merged**